### PR TITLE
Miniconda version

### PR DESCRIPTION
**Pull request by weaverba137**
This PR allows the miniconda version to be selected programmatically, and defaults to the currently (pinned) version.

### CODE REVIEW

File: travis/setup_conda_linux.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@
 # Install conda (http://conda.pydata.org/docs/travis.html#the-travis-yml-file)
 # Note that we pin the Miniconda version to avoid issues when new versions are released.
 # This can be updated from time to time.
-wget https://repo.continuum.io/miniconda/Miniconda3-4.5.4-Linux-x86_64.sh -O miniconda.sh
+if [[ -z "${MINICONDA_VERSION}" ]]; then
+    MINICONDA_VERSION=4.5.4
+    # MINICONDA_VERSION=latest
```
Comment (bsipocz) [inline travis/setup_conda_linux.sh]: no need for the comments, we'll wonder over time why they've been left here.

**Comment by weaverba137** [comment]
Will do...

**Comment by weaverba137** [comment]
Funny enough, `$MINICONDA_VERSION` is already implemented for appveyor.

**Comment by weaverba137** [comment]
So, both the Travis & Appveyor *scripts* set `MINICONDA_VERSION=4.5.4` by default.  However, the appveyor.yml file (but not the .travis.yml file) overrides this with `MINICONDA_VERSION=latest`.  Shouldn't the tests both use the same version?

**Comment by bsipocz** [comment]
@weaverba137 - I think it's OK to test multiple versions, it also double checks that the machinery works.

**Comment by weaverba137** [comment]
Good, thank you!

**Pull request merged**